### PR TITLE
Use pre-created travis virtualenv if applicable

### PR DESCRIPTION
**Pull request by joshfriend**
Saves a bit of time by not creating a second virtualenv within the Travis virtualenv.

### CODE REVIEW

**Comment by joshfriend** [comment]
I think we should stop trying to work around the `newrelic-plugin-agent` package weirdness. I haven't come across any more instances of this setup and I haven't made much progress in getting it fixed (MeetMe/newrelic-plugin-agent#383). I don't even use the package anymore TBH. If we get rid of it, we can do something like [this](https://github.com/flask-restful/flask-restful/blob/8bdba92ef54645ada501a39edc0bc68d34127b64/Makefile#L12).

tl;dr It's probably best to close this branch and start fresh if we want to do this.
